src/main/java/frc/robot/subsystems/tank/TankSubsystem.java

```diff
@@ -62,42 +64,53 @@ public void simulationPeriodic() {
   }
 
   /**
-   * Drive the system with the given power scales.
+   * Drive the system with the given power scales using the car system.
    * 
    * @param yScale       Scale in the forward/backward direction, from 1 to -1.
    * @param angularScale Scale in the rotational direction, from 1 to -1,
    *                     clockwise to counterclockwise.
    */
-  public void setDrivePowers(double yScale, double angularScale) {
-    setDrivePowers(yScale, angularScale, true);
+  public void setCarDrivePowers(double yScale, double angularScale) {
+    setCarDrivePowers(yScale, angularScale, true);
   }
 
-  public void setDrivePowers(double yScale, double angularScale, boolean squareInput) {
-
+  public void setCarDrivePowers(double yScale, double angularScale, boolean squareInput) {
     // Square the input if needed for finer control
     if (squareInput) {
       yScale = Math.copySign(yScale * yScale, yScale);
       angularScale = Math.copySign(angularScale * angularScale, angularScale);
     }
 
-    double leftPower = yScale + angularScale;
-    double rightPower = yScale - angularScale;
+    // Set motor output state
+    double leftPowerTemp = yScale + angularScale;
+    double rightPowerTemp = yScale - angularScale;
 
     // Scale powers greater than 1 back to 1 if needed
-    double largest_power = Math.max(Math.abs(leftPower), Math.abs(rightPower));
+    double largest_power = Math.max(Math.abs(leftPowerTemp), Math.abs(rightPowerTemp));
     if (largest_power > 1.0) {
       double scale = 1.0 / largest_power;
 
-      leftPower *= scale;
-      rightPower *= scale;
+      leftPowerTemp *= scale;
+      rightPowerTemp *= scale;
     }
 
-    // set state with new motor powers
-    this.yScale = leftPower;
-    this.angularScale = rightPower;
+    // Set motor output state
+    this.leftPower = leftPowerTemp;
+    this.rightPower = rightPowerTemp;
   }
 
-  public void followPathCommand() {
+  /**
+   * Drive the system with the given power scales using the tank system.
+   * 
+   * @param leftScale  Scale in the forward/backward direction of the left motor,
+   *                   from -1 to 1.
+   * @param rightScale Scale in the forward/backward direction of the right motor,
+   *                   from -1 to 1.
+   */
+  public void setTankDrivePowers(double leftScale, double rightScale) {
```

Comment:
Do we want to add a squareInput for this too, in case the joysticks need the same adjustment